stories/module-analytics-settings.stories.js

```diff
@@ -598,7 +598,7 @@ storiesOf( 'Analytics Module/Settings', module )
 			);
 		},
 		{
-			decorators: [ withRegistry ],
+			decorators: [ WithRegistry ],
```

Comment:
This is just a stray error that I caught looking at another issue. It was showing up in the terminal but not causing the story to fail the storybook tests probably because it caused all stories in the module to be invalid?